src/breadcrumb/breadcrumb-item.component.ts

```diff
@@ -28,4 +57,16 @@ export class BreadcrumbItemComponent {
 	@HostBinding("class.bx--breadcrumb-item--current") @Input() current = false;
 
 	@HostBinding("class.bx--breadcrumb-item") itemClass = true;
+
+	protected _href = "javascript:void(0)";
+
+	constructor(protected domSanitizer: DomSanitizer, @Optional() protected router: Router) { }
+
```

Comment:
Check if we are in a skeleton, have a href, or have a route

```suggestion

	useTemplate() {
		return this.skeleton || this._href || this.route;
	}
```